src/service/token-service.js

```diff
@@ -0,0 +1,66 @@
+const jwt = require("jsonwebtoken");
+const tokenModel = require("../models/token-model");
+
+class TokenService {
+  generateTokens(payload) {
+    const accesToken = jwt.sign(payload, process.env.JWT_ACCES_SECRET, {
+      expiresIn: "100m",
+    });
+
+    const refreshToken = jwt.sign(payload, process.env.JWT_REFRESH_SECRET, {
+      expiresIn: "30d",
+    });
+
+    return {
+      accesToken,
+      refreshToken,
+    };
+  }
+
+  validateAccessToken(token) {
+    try {
+      const userData = jwt.verify(token, process.env.JWT_ACCESS_SECRET);
```

Comment:
so JWT_ACCES_SECRET or process.env.JWT_ACCESS_SECRET? How is the variable called?